aea/configurations/base.py

```diff
@@ -1166,7 +1166,7 @@ class SkillConfig(ComponentConfiguration):
 
     default_configuration_filename = DEFAULT_SKILL_CONFIG_FILE
     package_type = PackageType.SKILL
-    configurable_fields = {"handlers", "behaviours", "models"}
+    configurable_fields = {"handlers", "behaviours", "models", "is_abstract"}
```

Comment:
@marcofavorito adding this here is not enough; how can this be extended so it is applied to the nested config?